addons/pos_restaurant/static/src/css/restaurant.css

```diff
@@ -315,3 +315,81 @@
     margin-right: 4px;
 }
 
+/* ------ SPLIT BILL SCREEN ------- */
+
+.splitbill-screen.screen .contents {
+    display: flex;
+    flex-flow: column nowrap;
+    margin: 0px auto;
+    max-width: 1024px;
+    border-left: dashed 1px rgb(215,215,215);
+    border-right: dashed 1px rgb(215,215,215);
+    height: 100%;
+}
+.splitbill-screen.screen .contents .main {
+    display: flex;
+    /* take the remaining vertical space */
+    flex: 1;
+    /* do not capture overflow in this element */
+    overflow: hidden;
+}
+.splitbill-screen.screen .main .lines {
+    display: flex;
+    flex: 1;
+    justify-content: center;
+    /* show scrollbar inside this element if its content overflows */
+    overflow-y: auto;
+}
+.splitbill-screen.screen .main .controls {
+    display: flex;
+    flex-flow: column nowrap;
+}
+.splitbill-screen.screen .pay-button {
+    margin: 16px;
+}
+.splitbill-screen.screen .pay-button .button {
+    background: #e2e2e2;
+    line-height: 74px;
+    font-size: 16px;
+    border: solid 1px rgb(202, 202, 202);
+    border-top-width: 0px;
+    cursor: pointer;
+    width: 100%;
+    text-align: center;
+}
+.splitbill-screen.screen .pay-button .button:first-child {
+    border-top-width: 1px;
+    border-top-left-radius: 3px;
+    border-top-right-radius: 3px;
+}
+.splitbill-screen.screen .pay-button .button:last-child {
+    border-bottom-left-radius: 3px;
+    border-bottom-right-radius: 3px;
+}
+.splitbill-screen.screen .pay-button .button:active {
+    background: black;
+    border-color: black;
+    color: white;
+}
+
+/* ------ WIDER SCREEN ------ */
+@media screen and (min-width: 768px) {
+    .splitbill-screen.screen .contents .main {
+        flex-flow: row nowrap;
```

Comment:
the flex flow here can be in the main section, no need to specify it in 'big screen'.

Also I am not sure you want to change the flex-direction. Probably you just want to 'wrap', when there is no pace to display elements side by side.

Reply:
A yes, that's what I want. I try that.

Reply:
https://drive.google.com/file/d/1glYmuIaku6e-hFqGQ_AL8mAlHufGWZFf/view
That is what I'm trying to accomplish.
1. when in wide screen, the 'order list' is in the left and the 'amount & pay button' at right. flex-flow is row and flex-grow is 1 so that both left and right have equal width.
2. when in narrow screen, the 'order list' is better on top, the 'amount & pay button' at bottom. to do this, flex-flow is column. (row wrap doesn't work, the elements are just stretching). Being stacked in column, the 'amount & pay button' should occupy the minimum height, so it should not flex.

I am able to get rid of the 'wider screen' media query.

Reply:
to make the wrap working, you should put a min-width on both elements, when there won't be place to guarantee the min-width it'll wrap. I think yyou can also use 'flex-basis', if you don't want a real min-width